src/openforms/js/compiled-lang/nl.json

```
@@ -6568,7 +6568,7 @@
   "uUf7cu": [
     {
       "type": 0,
-      "value": "Path relative to the Service API root."
+      "value": "Pad relatief aan de Service API."
```

```suggestion
      "value": "Pad relatief aan de Service API basisurl."
```
To be fair, though, I think my original label should include a 'url' at the end as well :)